components/write/WritingPanel.tsx

```diff
@@ -14,7 +14,7 @@ import { transitionPullV } from "../../utils/helpers/UiTransitionHelpers";
 import { scrollToTop } from "../../utils/hooks/RouteChangeHook";
 import { mainApi } from "../../utils/services/network/MainApi";
 import StatusPlaceholder from "../placeholder/StatusPlaceholder";
-import WritingPanelForm1 from "./WritingPanelForm";
+import WritingPanelForm from "./WritingPanelForm";
```

Comment:
Ok my bad, it was bugging

Reply:
Yea, i think it was making the form not executing the `entered` props from transition. I thought i was crazy, damn caches